From bd90903c5c0fd12b893ed330fae46170b1cb036c Mon Sep 17 00:00:00 2001
From: Kathiravan T <quic_kathirav@quicinc.com>
Date: Fri, 17 Mar 2023 15:00:14 +0530
Subject: [PATCH 154/281] soc: qcom: socinfo: Add IDs for IPQ5332 and its
 variant

Add SOC ID for Qualcomm IPQ5332 and IPQ5322 variants.

Change-Id: Idf12b53efb1f88e3b94ce5a98b1649869cc5dc51
Signed-off-by: Kathiravan T <quic_kathirav@quicinc.com>
---
 drivers/soc/qcom/socinfo.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/soc/qcom/socinfo.c b/drivers/soc/qcom/socinfo.c
index aa37e1bad095..620da318ea06 100644
--- a/drivers/soc/qcom/socinfo.c
+++ b/drivers/soc/qcom/socinfo.c
@@ -336,6 +336,8 @@ static const struct soc_id soc_id[] = {
 	{ 487, "SC7280" },
 	{ 495, "SC7180P" },
 	{ 507, "SM6375" },
+	{ 592, "IPQ5332" },
+	{ 593, "IPQ5322" },
 };
 
 static const char *socinfo_machine(struct device *dev, unsigned int id)
-- 
2.17.1

